channels/applicationinsights-channel-js/src/Statsbeat.ts

```diff
@@ -0,0 +1,203 @@
+import { NetworkStatsbeat } from "./NetworkStatsbeat";
+import { Sender } from "./Sender";
+import { StatsbeatCounter } from "./Constants";
+import { IConfig, Metric } from "@microsoft/applicationinsights-common";
+import { IConfiguration, IAppInsightsCore, IPlugin, ITelemetryPluginChain, ITelemetryItem, objKeys, dateNow,  } from '@microsoft/applicationinsights-core-js';
+import dynamicProto from '@microsoft/dynamicproto-js';
+
+export class Statsbeat {
+    public static INSTRUMENTATION_KEY = "c4a29126-a7cb-47e5-b348-11414998b11e";
+    public static STATS_COLLECTION_SHORT_INTERVAL: number = 900000; // 15 minutes
+    public static NETWORK = "Network";
+
+    constructor() {
+        let _networkCounter: NetworkStatsbeat;
+        let _sender: Sender;
+        let _handle: any;
+        let _statsbeatMetrics: { properties?: {} };
```

Comment:
Nit: Can you use `metrics` instead of `properties`?

Reply:
This is defining the "type" the actual value of _statsbeatMetric will be undefined